web-hosting-case-study.md

```diff
@@ -0,0 +1,121 @@
+---
+title: web hosting
+keywords: web hosting
+tags: web hosting
+sidebar: development-platforms
+permalink: development-platforms/example-product.html
+folder: development-platforms
+---
+
+## Introduction
+
+### Summarise the product's primary purpose in one or two paragraphs.
+
+Web hosting is a service that allows individuals and organizations to make their website accessible on the internet. It offer a range of services including email hosting, domain registration, and website management tools.
+
+## Brief History
+
+### Provide a brief history of the product, including major milestones or events.
+
+### You may use bullet points, numbered lists, paragraphs or a timeline.
+
+The concept of web hosting can be traced back to the early days of the internet in the 1990s, when the first websites were created and hosted on personal computers. As the internet grew in popularity, the need for more robust and reliable hosting solutions emerged.
+
+Some of the major milestones in the history of web hosting include:
+
+1993 - The first commercial web hosting company, Global Network Navigator, is launched
+
+1996 - The introduction of virtual hosting, which allows multiple websites to be hosted on a single server
+
+1998 - The launch of the first dedicated server hosting solutions
+
+2002 - The widespread adoption of cPanel, a popular website management tool
+
+2006 - The introduction of cloud hosting, which uses a network of servers to provide more flexibility and scalability
+
+2010 - The launch of platforms like Heroku and AWS Lambda, which allow for easy deployment and scaling of web applications
+
+2013 - The introduction of containerization technology like Docker, which allows for more efficient use of server resources
+As technology continues to advance, web hosting solutions have become more sophisticated and capable of meeting the needs of a wide range of users and websites.
+
+## Features
+
+### Outline the product's key features.
+
+N/A
```

Comment:
Instead of features you could write something about the hosting products that are available today?

Where could I look if I needed hosting?

Reply:
added a new paragraph